brainstormforce/ultimate-addons-for-gutenberg
# Fix: UAG Block - Device preview css not working in editor.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-437
- Block to be check
  1. Call To Action
  2. Inline Notice
  3. Info-Box
  4. Review Schema



### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop respectively block.
- Check responsive setting apply diff setting on diff devices and it should apply as per device in the editor.

### Checklist:
